app/server/controller/src/main/java/io/syndesis/server/controller/ControllersConfigurationProperties.java

```diff
@@ -29,6 +35,9 @@
     private int integrationStateCheckInterval = 60;
     private boolean exposeVia3scale;
 
+    @NestedConfigurationProperty
+    private CamelK camelk = new CamelK();
```

Comment:
Are this bits intended in this PR? The title is about `SyndesisHeaderStrategy` and they add a bit more than that.

Reply:
yes it is, as customizers are loaded using a property we may need this to tweak what customizers are loaded at runtime, now the header strategy customizer is included by default but it was not in the first iteration